tags/README.md

```diff
@@ -0,0 +1,145 @@
+# Tag component
+
+## Button design specs
+
+You can find the design specs on [www.decathlon.design](https://www.decathlon.design)
+
+## Usage with Vitamin Theme
+
+By inheriting the Vitamin theme in your app, you have the `VitaminTag` component.
+
+```kotlin
+implementation("com.decathlon.vitamin:tags:<version>")
+```
+
+```xml
+<com.decathlon.android.tags.VitaminTagAccent
+     android:layout_width="wrap_content"
+     android:layout_height="wrap_content"
+     android:text="Tag" />
+```
+
+As the tag component is inherited from chips, if you want to put an icon on your tag, you need to use this tag : `app:chipIcon`
+```xml
+<com.decathlon.android.tags.VitaminTagAccent
+     android:layout_width="wrap_content"
+     android:layout_height="wrap_content"
+     android:text="Tag"
+     app:chipIcon="@drawable/ic_vtmn_football_line" />
```

Comment:
The developer shouldn't know that the implementation of a tag is a `Chip`. We should have something like `tagIcon`.

Reply:
I created an attribute `tagIcon`